From 1e403f95d9c6773cea227538093541dc4dedf9b7 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Tue, 23 Mar 2021 13:24:50 +0200
Subject: [PATCH] [SW-0] gaudi: increase ppboot size

To introduce a generic reset decoding capability
that can work commonly for all platforms, gaudi1
ppboot needs to be marginally increased.
Therefore this patch increases the size by 1KB

Change-Id: Ia105e1baacbaecc957225a9248274f63141f6fe7
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi/common.h b/gaudi/common.h
index 374e28d..4ab14a4 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -7,7 +7,7 @@
 #ifndef COMMON_GAUDI_H_
 #define COMMON_GAUDI_H_
 
-#define PPBOOT_SIZE			0x4000		/* 16KB */
+#define PPBOOT_SIZE			0x4400		/* 17KB */
 #define PREBOOT_SIZE			0x20000		/* 128KB */
 #define SRAM_SIZE			0x1400000	/* 20MB */
 
